pkg/controller/windowsmachine/nodeconfig/nodeconfig.go

```diff
@@ -181,6 +181,11 @@ func (nc *nodeConfig) configureNetwork() error {
 	if err := nc.Windows.ConfigureKubeProxy(nc.node.GetName(), nc.node.Annotations[HybridOverlaySubnet]); err != nil {
 		return errors.Wrapf(err, "error starting kube-proxy for %s", nc.node.GetName())
 	}
+	// Start Windows metrics exporter service
+	if err := nc.Windows.ConfigureWindowsExporter(); err != nil {
```

Comment:
This call can go outside of network configurations after nc.configureNetwork(), since we are not making any changes to network here.

Reply:
Makes sense, thank you.